docs/developers/README.md

```diff
@@ -0,0 +1,11 @@
+# 💊 ReVanced Manager
+This documentation explains how [ReVanced Manager](https://github.com/ReVanced/revanced-manager) works.
```

Comment:
This is the developer documentation, it doesn't just necessarily explain how Manager works.

Reply:
Soon:tm: